wrapspawner/wrapspawner.py

```diff
@@ -184,6 +185,18 @@ class ProfilesSpawner(WrapSpawner):
             The first three values will be exposed in the input_template as {display}, {key}, and {type}"""
         )
 
+    @validate("profiles")
+    def _validate_profiles(self, proposal):
+        profiles = proposal.value
+
+        seen = set()
+        duplicated = {p[1] for p in profiles if p[1] in seen or seen.add(p[1])}
+        if len(duplicated):
+            logging.warning(
```

Comment:
If you do log, this can be:

```suggestion
            self.log.warning(
```

to use the current logger